src/test/unit_tests/test_namespace_fs.js

```diff
@@ -232,6 +232,113 @@ mocha.describe('namespace_fs', function() {
+            await fs_utils.file_must_exist(full_path_version_dir);
+            await fs_utils.create_fresh_path(dir1, 0o770);
+            await fs_utils.file_must_exist(dir1);
+            await fs_utils.create_fresh_path(dir1_version_dir, 0o770);
+            await fs_utils.file_must_exist(dir1_version_dir);
+            await fs_utils.create_fresh_path(dir2, 0o770);
+            await fs_utils.file_must_exist(dir2);
+            await fs_utils.create_fresh_path(dir2_version_dir, 0o770);
+            await fs_utils.file_must_exist(dir2_version_dir);
+            // export dir as a bucket
+            await rpc_client.pool.create_namespace_resource({
+                name: nsr,
+                nsfs_config: {
+                    fs_root_path: tmp_fs_root,
+                }
+            });
+            const obj_nsr = { resource: nsr, path: bucket_path };
+            await rpc_client.bucket.create_bucket({
+                name: bucket_name,
+                namespace: {
+                    read_resources: [obj_nsr],
+                    write_resource: obj_nsr
+                }
+            });
+            const policy = {
+                Version: '2012-10-17',
+                Statement: [{
+                    Sid: 'id-1',
+                    Effect: 'Allow',
+                    Principal: { AWS: "*" },
+                    Action: ['s3:*'],
+                    Resource: [`arn:aws:s3:::*`]
+                },
+            ]
+            };
+            // create accounts
+            let res = await nsfs_versioning.generate_nsfs_account({ admin: true });
+            s3_admin = nsfs_versioning.generate_s3_client(res.access_key, res.secret_key);
+            await s3_admin.putBucketPolicy({
+                Bucket: bucket_name,
+                Policy: JSON.stringify(policy)
+            }).promise();
+            // create nsfs account
+            res = await nsfs_versioning.generate_nsfs_account();
+            s3_client = nsfs_versioning.generate_s3_client(res.access_key, res.secret_key);
+            accounts.push(res.email);
+            await s3_client.putBucketVersioning({ Bucket: bucket_name, VersioningConfiguration: { MFADelete: 'Disabled', Status: 'Enabled' } }).promise();
+            const bucket_ver = await s3_client.getBucketVersioning({ Bucket: bucket_name }).promise();
+            assert.equal(bucket_ver.Status, 'Enabled');
+            await create_object(`${full_path}/${key}`, body, 'null');
+            await create_object(`${full_path_version_dir}/${version_key}`, version_body, 'null');
+            await create_object(`${dir1}/${key}`, body, 'null');
+            await create_object(`${dir1_version_dir}/${version_key}`, version_body, 'null');
+            await create_object(`${dir2}/${key}`, body, 'null');
+            await create_object(`${dir1_version_dir}/${version_key}`, version_body, 'null');
+        });
+
+        mocha.after(async () => {
+            //fs_utils.folder_delete(tmp_fs_root);
```

Comment:
please add it back for cleaning, can you please uncomment it in other tests in this file as well?

Reply:
that went unnoticed. Sure will do that